### ISSUE TITLE
Manifest needs to be updated for the requirements.{pip => txt} change

### ISSUE THREAD

**Issue by jmkacz** [amplify-education/terrawrap#185]
https://github.com/amplify-education/terrawrap/blob/main/MANIFEST.in#L2 needs to be changed from `*.pip` to `requirements*.txt`.

```
% pip install terrawrap==0.9.34
REDACTED
Collecting terrawrap==0.9.34
  Using cached terrawrap-0.9.34.tar.gz (41 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [8 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/private/var/folders/bl/mt7xf4ps37qb4ppyqj3wmncw0000gq/T/pip-install-b36i9rxd/terrawrap_2ce77ab6ba814200b2574268a199c1d4/setup.py", line 76, in <module>
          install_requires=get_requirements(),
        File "/private/var/folders/bl/mt7xf4ps37qb4ppyqj3wmncw0000gq/T/pip-install-b36i9rxd/terrawrap_2ce77ab6ba814200b2574268a199c1d4/setup.py", line 40, in get_requirements
          with open("requirements.txt", encoding="utf-8") as reqfile:
      FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

[notice] A new release of pip is available: 23.0.1 -> 24.0
[notice] To update, run: pip install --upgrade pip
```

**Comment by jmkacz** [comment]
```
% wget -O terrawrap-0.9.33.tar.gz "https://files.pythonhosted.org/packages/8a/97/970538864ed3b2f3cc546e5c8abd54e75f81d2693d55e6de5e9bcda5ec63/terrawrap-0.9.33.tar.gz#sha256=1f112fa9b95aef7b3065306b4e9049e12fb3ac647333320c15bbbb72ab1aa9d6"
% tar -tzf terrawrap-0.9.33.tar.gz | grep 'requirements\|\.txt\|\.pip'
terrawrap-0.9.33/requirements.pip    <====
terrawrap-0.9.33/terrawrap.egg-info/SOURCES.txt
terrawrap-0.9.33/terrawrap.egg-info/dependency_links.txt
terrawrap-0.9.33/terrawrap.egg-info/requires.txt
terrawrap-0.9.33/terrawrap.egg-info/top_level.txt
terrawrap-0.9.33/test-requirements.pip
```

```
% wget -O terrawrap-0.9.34.tar.gz "https://files.pythonhosted.org/packages/00/8b/88fd1f86173b38e91667293cbb7442522861a2a9633111e71d43559fffdc/terrawrap-0.9.34.tar.gz#sha256=fd24673e18655c84e7641067516838c506c7b3197ba12bf3ca8cd4104c6b816d"
% tar -tzf terrawrap-0.9.34.tar.gz | grep 'requirements\|\.txt\|\.pip'
terrawrap-0.9.34/terrawrap.egg-info/SOURCES.txt
terrawrap-0.9.34/terrawrap.egg-info/dependency_links.txt
terrawrap-0.9.34/terrawrap.egg-info/requires.txt
terrawrap-0.9.34/terrawrap.egg-info/top_level.txt
```

**Comment by kelibezhani** [comment]
This issue was addressed in PR #186
